src/utils/gfycat_sdk.ts

```diff
@@ -1,23 +1,24 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
-import Gfycat from 'gfycat-sdk';
+const GfycatConstructor = require('gfycat-sdk');
```

Comment:
This is a super weird solution to solve the typing problem, I would prefer to define an "full module any" type definition.